src/pages/book/record/select.tsx

```diff
@@ -1,40 +1,83 @@
+import { dehydrate, QueryClient, useQuery } from '@tanstack/react-query';
+import { GetServerSideProps } from 'next';
 import Image from 'next/image';
 import React from 'react';
 import { ReactElement } from 'react';
 import { useRecoilState } from 'recoil';
 
+import { getMyBookShelfApi } from '@/api/bookshelf';
 import AuthRequiredPage from '@/components/auth/AuthRequiredPage';
 import SelectModal from '@/components/book/record/SelectModal';
 import BookSelectLayout from '@/layout/BookSelectLayout';
-import { selectModalStateAtom } from '@/recoil/modal';
+import {
+  selectBookProps,
+  selectModalDataAtom,
+  selectModalStateAtom,
+} from '@/recoil/modal';
 import { NextPageWithLayout } from '@/types/layout';
 
 const BookRecordSelectPage: NextPageWithLayout = () => {
+  const { data, status } = useQuery(['getMyBookShelf', 'select'], () =>
+    getMyBookShelfApi(),
+  );
```

Comment:
p5;
인자가 없을 경우 `getMybookShelfApi` 로만 사용해도 될 것 같아요!
```suggestion
  const { data, status } = useQuery(['getMyBookShelf', 'select'], getMyBookShelfApi);
```

Reply:
오 이렇게 해도 되는군요